Added some improvements to the README. (#42)
DualCoder/vgpu_unlock: README.md

# vgpu\_unlock

Unlock vGPU functionality for consumer-grade Nvidia GPUs.

## Important!

> This tool is not guarenteed to work out of the box in some cases, so use it at your own risk.

Reviewer: ```suggestion
This tool is not guarenteed to work out of the box in some cases, so use it at
your own risk.
```
Keep lines below 80 characters (when reasonable).

## Description

This tool enables the use of Geforce and Quadro GPUs with the NVIDIA vGPU

> software. NVIDIA vGPU normally only supports a few datacenter Tesla and professional Quadro GPUs by design,
> but not consumer graphics cards through a software limitation. This tool aims to remove this limitation.  

Reviewer: ```suggestion
software. NVIDIA vGPU normally only supports a few datacenter Tesla and
professional Quadro GPUs by design, but not consumer graphics cards through a
software limitation. With vgpu\_unlock most Maxwell, Pascal, Volta (untested)
and Turing based GPUs can be used with vGPU (Ampere support is TODO).
```
I intended to put something like this here, what do you think?

> A community maintained Wiki written by Krutav Shah with a lot more information is [available here.](https://docs.google.com/document/d/1pzrWJ9h-zANCtyqRgS7Vzla0Y8Ea2-5z2HEi4X75d2Q/edit?usp=sharing)

Reviewer: ```suggestion
A community maintained Wiki written by Krutav Shah with a lot more information
is [available here.](https://docs.google.com/document/d/1pzrWJ9h-zANCtyqRgS7Vzla0Y8Ea2-5z2HEi4X75d2Q/edit?usp=sharing)
```
Line length

---
**NOTE**

> This script only works with graphics cards in the same generation as their professional Tesla counterparts.
> As a result, only Maxwell and newer generation Nvidia GPUs are supported. It is not designed to be used with
> low end graphics card models, so not all cards are guarenteed to work smoothly with vGPU. For the best experience,
> it is recommended to use graphics cards with the same chip model as the Tesla cards. The same applies to the
> operating system as well, as certain bleeding-edge Linux distributions may not work well with vGPU software.

Reviewer: ```suggestion
This script only works with graphics cards in the same generation as their
professional Tesla counterparts. As a result, only Maxwell and newer generation
Nvidia GPUs are supported. It is not designed to be used with low end graphics
card models, so not all cards are guarenteed to work smoothly with vGPU. For the
best experience, it is recommended to use graphics cards with the same chip
model as the Tesla cards. The same applies to the operating system as well, as
certain bleeding-edge Linux distributions may not work well with vGPU software.
```
Line length
